mod_reforged_AfterHooks/dynamic_spawns/unit_blocks/bandit_blocks.nut

```diff
@@ -33,7 +33,7 @@ local unitBlocks = [
 		ID = "UnitBlock.RF.BanditRanged",
 		DeterminesFigure = true,
 		UnitDefs = [
-			{ BaseID = "Unit.RF.BanditMarksmanLOW" },
+			{ BaseID = "Unit.RF.BanditPoacher" },
```

Comment:
Overall I think this may be _fine_. However, I am a bit afraid because currently we follow a very nice convention which also enables a nice level of "search" function through the codebase i.e. `Unit.RF.TroopID` convention. This change will break it. What are your thoughts about that?

Reply:
Okay makes sense to not break that.
I thought about a smaller scale alternative:
- Adding a comment mentioning Brigand Poacher after the definition of this unit
```
	{
		ID = "Unit.RF.BanditMarksmanLOW",	// Brigand Poacher
		Troop = "BanditMarksmanLOW",
		Figure = "figure_bandit_01", // Vanilla bandit poacher figure
	},
```